components/shooter.py

```diff
@@ -0,0 +1,61 @@
+from magicbot import tunable, feedback
+from rev import CANSparkMax
+from ids import SparkMaxIds, TalonIds
+import phoenix6
+from phoenix6.configs import (
+    config_groups,
+    MotorOutputConfigs,
+)
+from phoenix6.controls import VoltageOut
+import phoenix6.hardware
+
+
+class Shooter:
+    flywheel_speed = tunable(0.0)
+    inject_speed = tunable(0.0)
+
+    def __init__(self):
+        self.deployed = False
+        self.top_flywheel = phoenix6.hardware.TalonFX(TalonIds.top_flywheel)
+        self.bottom_flywheel = phoenix6.hardware.TalonFX(TalonIds.bottom_flywheel)
+        self.inject_flywheel = CANSparkMax(
+            SparkMaxIds.inject_flywheel, CANSparkMax.MotorType.kBrushless
+        )
+
+        # Configure bottom flywheel motor
+        bottom_config = self.bottom_flywheel.configurator
+        bottom_flywheel_config = MotorOutputConfigs()
+        bottom_flywheel_config.inverted = config_groups.InvertedValue.CLOCKWISE_POSITIVE
+
+        bottom_config.apply(bottom_flywheel_config)
+
+        self.inject_flywheel.setInverted(True)
+
+        self.should_inject = False
+
+    def shoot(self):
+        """causes shooter motors to spin"""
+        self.deployed = True
+
+    def inject(self):
+        self.should_inject = True
+
+    def is_ready(self):
+        return True
+
+    @feedback
+    def motor_speeds(self) -> float:
+        # return the current speed of both flywheel and inject motors
+        return self.flywheel_speed, self.inject_speed
```

Comment:
1. This currently echos the setpoints, so isn't very useful.
2. Per mypy the return type hint is incorrect. I intend on breaking such broken `@feedback` methods at some point.

```suggestion
```